src/artikler/dette-bor-du-vite/DetteBorDuViteEnglish.tsx

```diff
@@ -145,12 +145,14 @@ const DetteBorDuViteEnglish: React.FC = () => {
                 Your decision notice will include information about payments.
             </Avsnitt>
 
+            <Undertittel>Laws and regulations</Undertittel>
+
             <Avsnitt>
                 Read more about{" "}
-                <Lenke href="https://www.nav.no/en/Home/Benefits+and+services/Relatert+informasjon/financial-assistance-social-assistance">
-                    financial assistance
-                </Lenke>
-                .
+                <Lenke href="https://www.nav.no/no/person/flere-tema/sosiale-tjenester/relatert-innhold/lov-om-sosiale-tjenester-i-nav">
+                    Lov om sosiale tjenester i NAV
```

Comment:
Burde ikke 
`Lov om sosiale tjenester i NAV`  være på engelsk, selv om artikkelen det linkes til er på norsk? 
`Read more about Lov om sosiale tjenester i NAV `

Reply:
Jeg er litt usikker på om denne skal oversettes eller ikke. Selve loven heter jo "Lov om sosiale tjenester i arbeids- og velferdsforvaltningen (sosialtjenesteloven)", og har vel ikke en engelsk oversettelse.

Reply:
Kanskje vi bør undersøke hvordan andre deler av NAV referer til lovverket. Og om vi bør lenke til intern side på NAV, eller henvise direkte til lovverket (lovdata.no).

Reply:
Jeg tenker bare det er rart om jeg hadde lest en nettside til et annet land og det sto: 
`Read more about kldsjhue huehfe cusu iwjio. 
(Only available in Xxxxx)."

Da er det vanskelig å forstå hva man går glipp av

Reply:
Svar fra fag angående oversettelse av engelsk. "Siden vi lenker til finnes ikke på engelsk. Tror ikke vi skal oversette navnet på loven. Det var ikke gjort på faktaark siden i hvertfall". Så lar den stå slik inntil videre.